graphql/types.lua

```diff
@@ -299,9 +299,7 @@ end
 local function coerceLong(value)
   if value ~= nil then
     value = tonumber64(value)
-    if not isLong(value) then return end
```

Comment:
It's quite strange logic. I guess we could move isLong at this function start and raise an error if condition is wrong. Currently user could return even string instead of number and in some cases (if this string contains numeric values) user gets correct result.

Reply:
Seems that this must be fxed not only in this function but everywhere in types submodule. Filed a new issue: https://github.com/tarantool/graphql/issues/56

Reply:
Actually isValueOfTheType called from validate* routines and there if isValueOfTheType fails error is raised. We can raise error in isValueOfTheType but this leads to use pcall in validate* routines because we have to return meanningfull error and in isValueOfTheType we don't have enougth context info: variable or field name, etc.